hack/config-default.sh

```diff
@@ -15,6 +15,7 @@ kubevirtci_git_hash="2107080951-4492ebe"
 conn_check_ipv4_address=${CONN_CHECK_IPV4_ADDRESS:-""}
 conn_check_ipv6_address=${CONN_CHECK_IPV6_ADDRESS:-""}
 conn_check_dns=${CONN_CHECK_DNS:-""}
+apply_default_e2e_configuration=${APPLY_DEFAULT_E2E_CONFIGURATION:-true}
```

Comment:
Are you sure we need another env varisable? We have `FUNC_TEST_ARGS` where you can pass additional flags.

Reply:
@rmohr Hi. `FUNC_TEST_ARGS` is for the Gikngo flags, `apply_default_e2e_configuration` flag is from our test suite. You can see in `hack/functests.sh` that in the `functest` function the assembly of `extra_flags` is enclosed in the function w/o any option to append extra flags from the outside ENV. I am all for doing all this pre-configuration in a declarative manner.

Reply:
They get applied to the same binary at the end. Maybe I miss something but ginkgo flags are just a subset of flags which you can apply to our functional test suite.

Reply:
I've tried to `export FUNC_TEST_ARGS="-apply_default_e2e_configuration=false"` and then `make functest` and it failed because ginkgo didn't recognize the flag.
```code
function functest() {
    extra_args="${extra_args} -apply-default_e2e_confguration ...
...
...
    _out/tests/ginkgo -r --slowSpecThreshold 60 $@ \ # <---- here is where FUNC_TEST_ARGS evaluated
      _out/tests/tests.test -- ${extra_args}  ...
....
}
```

Reply:
Ok, now I see. That was not always like that. How about adding another general global env variables to restore the old flexibility? Having an env variables per flag does not scale well.

Reply:
@rmohr Sounds reasonable. I've made `extra_args` public, changed it's name to `EXTRA_ARGS`. Now if we don't want to use the default kv e2e configuration we can do `export EXTRA_ARGS="-apply-default-e2e-configuration=false"` and it will work

Reply:
Hm. I think we would have to deprecate `FUNC_TEST_ARGS` to allow better names and I am not sure if we want to do that right now, but good names would be:

```
KUBEVIRT_FUNC_TEST_GINKGO_ARGS
KUBEVIRT_FUNC_TEST_SUITE_ARGS
```

Maybe if we set `KUBEVIRT_FUNC_TEST_GINKGO_ARGS` to `FUNC_TEST_ARGS` we could transition right now.

Reply:
@rmohr Yes I see what you mean. Let's deprecate `FUNC_TEST_ARGS` in a seperate PR since it spans across multiple files and it will take this PR a little bit out of the scope. PTAL at the new changes.